ci/deb-ppa-upload/Dockerfile

```diff
@@ -1,20 +1,35 @@
-FROM ubuntu:20.04
+FROM ubuntu:20.04 as base
 SHELL ["/bin/bash", "-e", "-o", "pipefail", "-u", "-x", "-c"]
 
 # See https://github.com/phusion/baseimage-docker/issues/58 for why DEBIAN_FRONTEND=noninteractive is needed.
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y debhelper devscripts fakeroot dh-python gawk python3-all python3-paramiko python3-setuptools \
     && rm -rfv /var/lib/apt/lists/*
 
+FROM base AS CIRCLE_CI
```

Comment:
Another nit: so far I've rather seen Dockerfile stages being named in lowercase... `CIRCLE_CI` looks as is it were an env var